tamr_unify_client/dataset/resource.py

```diff
@@ -84,6 +84,25 @@ def _update_records(self, updates, **json_args):
             .json()
         )
 
+    def upsert_from_dataframe(self, df, primary_key_name, ignore_nan=True):
```

Comment:
```suggestion
    def upsert_from_dataframe(self, df, *, primary_key_name, ignore_nan=True):
```
^ to enforce keyword-specified arguments after `df`